<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetColorEffect</class>
 <widget class="QWidget" name="WidgetColorEffect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_38">
     <item row="0" column="0" colspan="2">
      <widget class="QFrame" name="mShadowFrame">
       <layout class="QGridLayout" name="gridLayout_7" columnstretch="0,0,0">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="2" column="2">
         <widget class="QgsSpinBox" name="mSaturationSpinBox">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="decimals" stdset="0">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="mColorizeCheck">
          <property name="text">
           <string>Colorize</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QSlider" name="mTranspSlider">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="pageStep">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSlider" name="mSliderColorizeStrength">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="mSliderContrast">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>0</height>
           </size>
          </property>
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Contrast</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Brightness</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSlider" name="mSliderSaturation">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>0</height>
           </size>
          </property>
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QgsSpinBox" name="mContrastSpinBox">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="mSliderBrightness">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>0</height>
           </size>
          </property>
          <property name="minimum">
           <number>-255</number>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
          <property name="tickInterval">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelSaturation">
          <property name="text">
           <string>Saturation</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QSpinBox" name="mColorizeStrengthSpinBox">
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>Transparency</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_30">
          <property name="text">
           <string>Blend mode</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1" colspan="2">
         <widget class="QgsEffectDrawModeComboBox" name="mDrawModeComboBox"/>
        </item>
        <item row="7" column="1" colspan="2">
         <widget class="QgsBlendModeComboBox" name="mBlendCmbBx"/>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Draw mode</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QgsDoubleSpinBox" name="mTranspSpnBx">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="suffix">
           <string> %</string>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QgsSpinBox" name="mBrightnessSpinBox">
          <property name="minimum">
           <number>-255</number>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QgsColorButtonV2" name="mColorizeColorButton">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="labelGrayscale">
          <property name="text">
           <string>Grayscale</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1" colspan="2">
         <widget class="QComboBox" name="mGrayscaleCombo"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsColorButtonV2</class>
   <extends>QToolButton</extends>
   <header>qgscolorbuttonv2.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsSpinBox</class>
   <extends>QSpinBox</extends>
   <header>qgsspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsBlendModeComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsblendmodecombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsEffectDrawModeComboBox</class>
   <extends>QComboBox</extends>
   <header>qgseffectdrawmodecombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mSliderBrightness</tabstop>
  <tabstop>mBrightnessSpinBox</tabstop>
  <tabstop>mSliderContrast</tabstop>
  <tabstop>mContrastSpinBox</tabstop>
  <tabstop>mSliderSaturation</tabstop>
  <tabstop>mSaturationSpinBox</tabstop>
  <tabstop>mColorizeCheck</tabstop>
  <tabstop>mSliderColorizeStrength</tabstop>
  <tabstop>mColorizeStrengthSpinBox</tabstop>
  <tabstop>mColorizeColorButton</tabstop>
  <tabstop>mGrayscaleCombo</tabstop>
  <tabstop>mTranspSlider</tabstop>
  <tabstop>mTranspSpnBx</tabstop>
  <tabstop>mBlendCmbBx</tabstop>
  <tabstop>mDrawModeComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mSliderBrightness</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mBrightnessSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mBrightnessSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mSliderBrightness</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mSliderContrast</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mContrastSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mContrastSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mSliderContrast</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mSliderSaturation</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mSaturationSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mSaturationSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mSliderSaturation</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mSliderColorizeStrength</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mColorizeStrengthSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mColorizeStrengthSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mSliderColorizeStrength</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
